pymeasure/instruments/instrument.py

```diff
@@ -101,6 +102,35 @@ def id(self):
         else:
             raise NotImplementedError("Non SCPI instruments require implementation in subclasses")
 
+    @property
+    def quick_properties_list(self):
+        return self.quick_props_list
+
+    @quick_properties_list.setter
+    def quick_properties_list(self, prop_list):
+        if type(prop_list) is str:
+            self.quick_props_list = [prop_list]
+        elif type(prop_list) is list:
+            self.quick_props_list = prop_list
+        else:
+            raise TypeError("Expected input of type str or list")
+
+    @property
+    def quick_properties(self):
+        prop_dict = {}
```

Comment:
You can document the behaviour of the new property by adding a relevant docstring to this getter. You can also explain the relationship to `self.quick_properties_list`.